api/src/test/java/org/projectnessie/tools/catalog/migration/api/ITNessieToHiveCatalogMigrator.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.catalog.migration.api;
+
+import java.util.Collections;
+import java.util.Set;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.tools.catalog.migration.api.test.HiveMetaStoreRunner;
+
+public class ITNessieToHiveCatalogMigrator extends AbstractTestCatalogMigrator {
+
+  @BeforeAll
+  protected static void setup() throws Exception {
+    HiveMetaStoreRunner.startMetastore();
+
+    initializeSourceCatalog(CatalogMigrationUtil.CatalogType.NESSIE, Collections.emptyMap());
+    targetCatalog = HiveMetaStoreRunner.hiveCatalog();
+
+    createNamespaces();
+  }
+
+  @AfterAll
```

Comment:
`dropNamespaces()` has been moved to `tearDown` of parent class `AbstractTestCatalogMigrator` and `AbstractCLIMigrationTest` to avoid calling in each testcase. 

But for the IT testcases that involve hive, `HiveMetaStoreRunner` will be stopped before calling the parent class `tearDown`. Hence, overriding the tearDown for hive IT testcases to call `dropNamespaces()` first.